### DISCUSSION TITLE
Cannot generate QRCODE

### DISCUSSION THREAD

**Discussion by Rafucho25** [zepp-health/zeppos-forum#44] [Developer Feedback]
Hi, as the title says I try to generate a qr code but the screen does not show anything, I copy the exact code example from the documentation [here](https://docs.zepp.com/docs/1.0/reference/device-app-api/hmUI/widget/QRCODE/) this happen both the simulator and the device. The device is amazfit band 7.

**Comment by htoooth** [comment]
Hi @Rafucho25, 
sorry, qrcode is not available in amazfit band7.
